webapp/components/notifications/Notification/Notification.vue

```diff
@@ -42,11 +51,26 @@ export default {
   },
   computed: {
     excerpt() {
-      return this.$filters.removeLinks(this.post.contentExcerpt)
+      const excerpt = this.post.id ? this.post.contentExcerpt : this.comment.contentExcerpt
+      return (
+        (!this.post.id ? '<b>Comment: </b>' : '') + excerpt.replace(/<(?:.|\n)*?>/gm, '').trim()
+      )
     },
     post() {
       return this.notification.post || {}
     },
+    comment() {
+      return this.notification.comment || {}
+    },
+    postParams() {
```

Comment:
Well, it's not `postParams` anymore right? Just `params` maybe?

Reply:
Yes, you are right 🤗
Renamed this …